persimmon/website/views.py

```diff
@@ -342,6 +345,39 @@ def reset_password_sent(request):
 
 
 @api_function
+def schedule(request, time: str):
+    user = current_user(request, expect_not_logged_in=False)
+    for empteller in User.objects.get(employee_level=1):
+        if not Appointment.objects.get(employee=empteller, time=time):
+            newapp = Appointment.objects.create(user, empteller, time)
+            newapp.save()
+    return {}
```

Comment:
The main issue here is that this will schedule _every_ available teller at the given time. You want a `return {}` after `newapp.save()`.

Additionally, you should change the final `return {}`, which now corresponds to an error condition, to `return {"error": "No employees available at this time"}`

Reply:
Additionally, your usage of `get` is wrong here - you want `filter`. `get` is used to return exactly one object, and it will error if it would return zero or more than one. Also, I think you want `.filter().exists()` for the second one. You can't test truthiness on query objects directly.